src/components/PostEditor/PostEditor.js

```diff
@@ -2,18 +2,15 @@
 import PropTypes from 'prop-types'
```

Comment:
Some linting, but substantial updates related to new editor. Also some refactoring to the `TopicsSelector` and "post.topics" handling in general within PostEditor. All needed to work well with new editor.